pkg/virt-operator/kubevirt_test.go

```diff
@@ -1487,8 +1616,14 @@ var _ = Describe("KubeVirt Operator", func() {
 			// the cluster from newer versions when an older version is being deployed.
 			// On create this prevents invalid specs from entering the cluster
 			// while controllers are available to process them.
-			Expect(totalPatches).To(Equal(patchCount - 2))
+
+			// 3 because 2 for virt-controller service and deployment
+			// and
+			// 1 because of the pdb of virt-controller
+			Expect(totalPatches).To(Equal(patchCount - 3))
 			Expect(totalUpdates).To(Equal(updateCount))
+
+			Expect(resourceChanges["poddisruptionbudgets"][Patched]).To(Equal(1))
```

Comment:
nice to track changes in more detail than before :+1:

Reply:
Yeah actually I needed this myself to understand more about what was going on. In a further step we might use this to increase the expectation granularity.